bcrypt.js

```diff
@@ -155,8 +164,16 @@ module.exports.compare = function(data, hash, cb) {
         });
     }
 
-    if (!cb || typeof cb !== 'function') {
-        return;
+    //cb exists but not a function
```

Comment:
nitpick: space after the `//` I think that's how we do it in other places in the file.